### PR TITLE
3.0: Change tag prefix to 'parallelcluster:'

### PR DESCRIPTION
**Pull request by rexcsn**
* Change tag prefix from 'aws-parallelcluster-' to 'parallelcluster:'

Signed-off-by: Rex <shuningc@amazon.com>

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ This file is used to list changes made in each version of the aws-parallelcluste
 **CHANGES**
 
 - Drop support for SGE and Torque schedulers.
+- Change tags prefix from `aws-parallelcluster-` to `parallelcluster:`
```
Comment (enrico-usai) [inline CHANGELOG.md:12]: nit: final dot.

File: src/slurm_plugin/common.py [hunk 2]
```diff
@@ -378,7 +378,7 @@ def get_cluster_instances(self, include_head_node=False, alive_states_only=True)
         if alive_states_only:
             args["Filters"].append({"Name": "instance-state-name", "Values": list(EC2_INSTANCE_ALIVE_STATES)})
         if not include_head_node:
-            args["Filters"].append({"Name": "tag:aws-parallelcluster-node-type", "Values": ["Compute"]})
+            args["Filters"].append({"Name": "tag:parallelcluster:node-type", "Values": ["Compute"]})
```
Comment (enrico-usai) [inline src/slurm_plugin/common.py:381]: I think we should align other tags too. e.g. tag:ClusterName but we can do it in a separate patch.

File: CHANGELOG.md (hunk 1 above)
Comment (rexcsn) [reply CHANGELOG.md:12]: Done

**Review by enrico-usai — APPROVED** [review]
